network/src/commonMain/kotlin/com/wire/kalium/network/api/user/register/NewUser.kt

```diff
@@ -38,7 +42,7 @@ internal data class ActivationRequest(
 
 
 @Serializable
-internal data class RegisterRequest(
+internal data class NewUser(
```

Comment:
```suggestion
internal data class NewUserRequest(
```

or, if we use it for both sending and receiving:

```suggestion
internal data class NewUserDTO(
```